src/msg/Message.h

```diff
@@ -221,9 +221,9 @@ class Message : public RefCountedObject {
   /* time at which message was fully read */
   utime_t recv_complete_stamp;
 
-  ConnectionRef connection;
+  ConnectionRef connection = nullptr;
```

Comment:
see http://www.boost.org/doc/libs/1_61_0/libs/smart_ptr/intrusive_ptr.html#constructors

`ConnectionRef` is reset by default.